manifests/config.pp

```diff
@@ -47,7 +47,7 @@
     }
   }
 
-  if $::vault::install_method == 'archive' {
+  if $::vault::install_method == 'archive' or $::vault::manage_service_file {
```

Comment:
@jsok I wanted to preserve the current implementation as much as possible.
But yeah, now with the rspec tests in place I can add that possibility and keep the same defaults.